pylearn2/devtools/tests/test_format.py

```diff
@@ -586,22 +586,18 @@
     'datasets/tests/test_npy_npz.py',
     'linear/tests/test_conv2d.py',
     'devtools/tests/pep8/pep8.py',
-    'devtools/tests/pep8/__init__.py']
-
-# add files which have long execution time to whitelist_docstrings
-whitelist_docstrings.extend([
-    'sandbox/cuda_convnet/debug.py',
-    'energy_functions/tests/test_rbm_energy.py',
+    'devtools/tests/pep8/__init__.py'
+    'scripts/lcc_tangents/make_dataset.py',
     'scripts/icml_2013_wrepl/multimodal/make_wordlist.py',
-    'scripts/make_weights_image.py', 'costs/ebm_estimation.py',
-    'classifier.py', 'scripts/lcc_tangents/make_dataset.py',
-    'scripts/datasets/make_cifar10_whitened.py',
-    'scripts/datasets/make_cifar100_patches.py',
-    'scripts/datasets/make_cifar10_gcn_whitened.py',
+    'scripts/datasets/make_stl10_whitened.py',
     'scripts/datasets/make_stl10_patches_8x8.py',
+    'scripts/datasets/make_stl10_patches.py',
+    'scripts/datasets/make_cifar10_gcn_whitened.py',
```

Comment:
Now that you have added `cifar10_gcn_whitened`, it complains about `cifar10_whitened`, so in the end you actually need both.